R/download_checkpoint.R

```diff
@@ -16,26 +16,50 @@
 
 #' Download a BERT checkpoint
 #'
-#' Downloads the specified BERT checkpoint from the Google Research collection.
+#' Downloads the specified BERT checkpoint from the Google Research collection
+#' or other repositories.
+#'
+#' @section Checkpoints: `download_BERT_checkpoint` knows about several
+#'   pre-trained BERT checkpoints. You can specify these checkpoints using the
+#'   `model` parameter. Alternatively, you can supply a direct `url` to any BERT
+#'   tensorflow checkpoint.
+#'
+#'   \tabular{rccccl}{ model \tab layers \tab hidden \tab heads \tab parameters
+#'   \tab special\cr bert_base_* \tab 12 \tab 768 \tab 12 \tab 110M\cr
+#'   bert_large_* \tab 24 \tab 1024 \tab 16 \tab 340M\cr bert_large_*_wwm \tab
+#'   24 \tab 1024 \tab 16 \tab 340M \tab whole word masking\cr
+#'   bert_base_multilingual_cased \tab 12 \tab 768 \tab 12 \tab 110M \tab 104
+#'   languages\cr bert_base_chinese \tab 12 \tab 768 \tab 12 \tab 110M \tab
+#'   Chinese Simplified and Traditional\cr scibert_scivocab_* \tab 12 \tab 768
+#'   \tab 12 \tab 110M \tab Trained using the full text of 1.14M scientific
+#'   papers (18\% computer science, 82\% biomedical), with a science-specific
+#'   vocabulary.\cr scibert_basevocab_uncased \tab 12 \tab 768 \tab 12 \tab 110M
+#'   \tab As scibert_scivocab_*, but using the original BERT vocabulary. }
 #'
 #' @param model Character vector. Which model checkpoint to download.
-#' @param dir Character vector. Destination directory for checkpoint. Leave
+#' @param dir Character vector. Destination directory for checkpoints. Leave
 #'   `NULL` to allow `RBERT` to automatically choose a directory. The path is
```

Comment:
I recall that preferred style for typesetting package names is to use regular font.